### PR TITLE
Working

### PR DESCRIPTION
**Pull request by ali-tab**
# Ticket OR Documentation Changes
Replace this with the Jira link and choose the correct title for this section. If `Documentation Change` delete this text instead.

# Description
Please include a small summary of the change (nothing too in-depth as the attached ticket should have more detail)

# Type of change
Delete options that are not relevant. Can be an unnecessary section

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Tests
Please describe the tests that you ran to verify your changes.

# Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

File: src/client/src/router.js [hunk 1]
```diff
@@ -34,6 +36,14 @@ const router = createBrowserRouter([
     path: '/group/join',
     element: <JoinGroup />,
   },
+  {
+    path: '/social',
+    element: <Social />,
+  },
+  {
+    path: '/Preferences',
```
Comment (TAsif) [inline src/client/src/router.js]:
```suggestion
    path: '/preferences',
```

File: src/client/src/pages/Social.jsx [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+
+  const handleCreateGroupClick = (e) => {
+    e.preventDefault();
+    navigate('/group/create');
+  };
+
+
+    return (
+
+<Box>
```
Comment (TAsif) [inline src/client/src/pages/Social.jsx]: The formatting on this is funky, Prettier/VS Code has autoformat which you should run - if its not working its fine, when it's merged I'll refactor it

File: src/client/src/pages/Social.jsx [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+            onClick={handleCreateGroupClick}>
+            Create Group
+          </Button>
+          <Button
+            variant="outlined"
+            size="large"
+            onClick={handleJoinGroupClick}>
+            Join Group
+          </Button>
+        </Box>
```
Comment (TAsif) [inline src/client/src/pages/Social.jsx]: We shouldn't have navigate from social to create group, it should navigate back home instead and from there a user can go to those pages

File: src/client/src/pages/Social.jsx [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+      </Box>
+    </Box>
+
+
+
+
+    );
+  };
+  
+  export default Social;
```
Comment (TAsif) [inline src/client/src/pages/Social.jsx]: This page needs an "Add Friends" button before merge, we can deal with "Delete Friend" afterwards

**Review by TAsif — CHANGES_REQUESTED** [review]

**Review by madhi-naga — APPROVED** [review]

**Review by TAsif — APPROVED** [review]

**Pull request merged**